operators.py

```diff
@@ -3055,13 +3077,24 @@ class VIEW3D_OT_slvs_add_angle(
     value: FloatProperty(
         name="Angle", subtype="ANGLE", unit="ROTATION", options={"SKIP_SAVE"}
     )
-    setting: BoolProperty(name="Invert")
+    old_setting: BoolProperty(default = False, options={"SKIP_SAVE"})
+    setting: BoolProperty(name="Invert", default = False, get=invert_angle_getter, set=invert_angle_setter)
```

Comment:
Instead of using getter/setter I tested with using update and it works great.

If you want to continue with getter/setter, maybe you could use self.properties.is_property_set("settings") to detect the first time?